docs/Materials.md.html

```diff
@@ -858,6 +879,30 @@
 }
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+### postLightingBlending
+
+Type
+:    `string`
+
+Value
+:     Any of `opaque`, `transparent`, `add`. Defaults to `opaque`.
```

Comment:
Would defaulting to "transparent" or "add" make more sense? To me, opaque seems like the least useful of the 3.

Reply:
Yeah I've been debating that, the main reason I chose opaque as the default was to be consistent with the other blending option. It's also obvious what happens by default. Not sure which of transparent or add would be the better default.

Reply:
I'll switch the default to transparent.

Reply:
👍 sounds good